code/datums/goon_mutations_disabilities.dm

```diff
@@ -143,7 +143,7 @@
 		message = replacetext(message,"i don't know","wot mate")
 		message = replacetext(message,"no","naw")
 		message = replacetext(message,"robust","chin")
-		message = replacetext(message,"hi","how what how")
+		message = replacetext(message," hi ","how what how")
```

Comment:
Looking at this now, you didn't even add spaces to the other part.
So the message ```"Why hi there!"``` would become ```"Whyhow what howthere!"```.